src/lib/ppx_coda/tests/dune

```diff
@@ -8,45 +8,45 @@
 ;; expiration
 (library
   (name unexpired)
-  (preprocess (pps ppx_jane ppx_deriving_yojson ppx_coda))
+  (preprocess (pps ppx_jane ppx_deriving_yojson ppx_version))
   (modules unexpired))
 
 ;; version syntax
 (library
   (name good_version_syntax)
-  (preprocess (pps ppx_jane ppx_deriving_yojson ppx_coda))
+  (preprocess (pps ppx_jane ppx_deriving_yojson ppx_version))
```

Comment:
This (and the other versioning tests) should probably live in `ppx_version`

Reply:
in a subsequent PR